CRM/Civioffice/DocumentRendererType/LocalUnoconv.php

```diff
@@ -212,9 +212,10 @@ public function validateSettingsForm(CRM_Civioffice_Form_DocumentRenderer_Settin
         $unoconv_binary_path = $form->_submitValues['unoconv_binary_path'];
         $unoconv_lock_file_path = $form->_submitValues['unoconv_lock_file_path'];
 
-        if (!file_exists($unoconv_binary_path)) {
-            $form->_errors['unoconv_binary_path'] = E::ts("File does not exist. Please provide a correct filename.");
-        }
+        // We can't check eg. /usr/bin/unoconv on a site with open_basedir restrictions in place so this check would always fail
+        //if (!file_exists($unoconv_binary_path)) {
+        //    $form->_errors['unoconv_binary_path'] = E::ts("File does not exist. Please provide a correct filename.");
+        //}
```

Comment:
Lets remove the check altogether and add a more elaborate comment.
```suggestion
        // There used to be a file_exists() check here for validating that the unoconv binary exists in the given path.
        // We can't however check eg. /usr/bin/unoconv on a site with open_basedir restrictions in place as this check
        // would always fail. There is a check running `unoconv --version` in the isReady() method() which implicitly
        // covers the validation of the unconv binary being accessible.
```

Reply:
Thanks @jensschuppe I accepted your change and squashed commits